keyboards/mechlovin/olly/jf/config.h

```diff
@@ -21,18 +21,3 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* key matrix size */
 #define MATRIX_ROWS 6
 #define MATRIX_COLS 19
```

Comment:
please move these two settings to a, new, `config.h` file inside `keyboards/mechlovin/olly/jf/rev1` directory
```suggestion
```